src/cli.ts

```diff
@@ -17,8 +17,16 @@ import { CommandError, SSEConnectionError, UserCancelledError } from './errors'
 import log from './logger'
 import tree from './modules/tree'
 import notify from './update'
+import * as semver from 'semver'
+import * as os from 'os'
 import { isVerbose, VERBOSE } from './utils'
 
+const nodeVersion = process.version.replace('v', '')
+if (!semver.satisfies(nodeVersion, pkg.engines.node)) {
```

Comment:
Once we are getting the required version from the `pkg.engines.node` I would suggest updating the message dynamically depending on the required minimum version. Otherwise, if we change the expected version from 10 to 12 and forget to update the message, the user may get confused if he already has version 10 installed.

What do you think?

Reply:
Oops, you're right